### PR TITLE
DDT-10

### PR DESCRIPTION
**Pull request by Abdiiir**
- Tests for OVS_API_2.0.0
- JSON Schema for OVS_API_2.0.0
- OVS_V2 & TNT_V2 created.

### CODE REVIEW

File: .github/workflows/validateOVSV1X.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Run testsuite on OVS master
```
Comment (HenrikHL) [inline .github/workflows/validateOVSV1X.yml:1]:
```suggestion
name: Run testsuite on OVS v1.X branch
```

File: src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+                contentType("application/json").
+                body(map).
+                post(Configuration.ROOT_URI + "/timestamps").
+                then().
+                assertThat().
+                statusCode(400);
+    }
+
+    @Test
+    public void testEventLocationQueryParam() {
```
Comment (HenrikHL) [inline src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java:118]: I think this test name is wrong - as it is not testing query parameters. It is testing eventLocation field in a timestamp

File: src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+                contentType("application/json").
+                body(map).
+                post(Configuration.ROOT_URI + "/timestamps").
+                then().
+                assertThat().
+                statusCode(400);
+    }
+
+    @Test
+    public void testVesselPositionQueryParam() {
```
Comment (HenrikHL) [inline src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java:174]: I think this test name is wrong - as it is not testing query parameters. It is testing vesselPosition field in a timestamp

File: src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+                contentType("application/json").
+                body(map).
+                post(Configuration.ROOT_URI + "/timestamps").
+                then().
+                assertThat().
+                statusCode(400);
+    }
+
+    @Test
+    public void testModeOfTransportQueryParam() {
```
Comment (HenrikHL) [inline src/test/java/org/dcsa/api_validator/ovs/v2/PostTimestampsTest.java:229]: I think this test name is wrong - as it is not testing query parameters. It is testing modeOfTransport field in a timestamp

File: src/test/resources/ovs/v2/EventSchema.json [hunk 5]
```diff
@@ -280,7 +280,7 @@
         },
         "publisher": {
           "description": "refers to a company or a legal entity.",
-          "type": "object",
+          "type": ["array","null","object"],
```
Comment (HenrikHL) [inline src/test/resources/ovs/v2/EventSchema.json:283]: Does this mean that arrays are supported? ...because they don't ;-)

Comment (Abdiiir) [reply src/test/resources/ovs/v2/EventSchema.json:283]: I agree, will fix this now.

File: .github/workflows/validateOVSMaster.yml [hunk 6]
```diff
@@ -1,40 +1,52 @@
-name: Run testsuite on OVS master
+name: Run testsuite on OVS aligning_to_updated_IM
```
Comment (HenrikHL) [inline .github/workflows/validateOVSMaster.yml:1]:
```suggestion
name: Run testsuite on OVS master
```

File: .github/workflows/validateOVSMaster.yml [hunk 7]
```diff
@@ -1,40 +1,52 @@
...
       with:
-        repositories: '[{ "id": "github", "name": "DCSA Backend repo", "url": "https://maven.pkg.github.com/dcsaorg/DCSA-Core", "releases": { "enabled": "false" }, "snapshots": { "enabled": "true" } }]'
+        repositories: '[{ "id": "github", "name": "DCSA Backend repo", "url": "https://maven.pkg.github.com/dcsaorg/DCSA-Core", "releases": { "enabled": "true" }, "snapshots": { "enabled": "true" } }]'
         servers: '[{ "id": "github", "username": "${{ secrets.DCSA_USER }}", "password": "${{ secrets.DCSA_PACKAGES_PAT }}" }]'    
     - name: Fetch backend
       uses: actions/checkout@v2
       with:
         repository: dcsaorg/DCSA-OVS
-        ref: master
+        ref: aligning_to_updated_IM
```
Comment (HenrikHL) [inline .github/workflows/validateOVSMaster.yml:22]:
```suggestion
        ref: master
```

**Review by mp-assco — APPROVED** [review]

**Pull request merged**